src/index.html

```diff
@@ -7,6 +7,40 @@
     <link rel="stylesheet" href="./styles/main.scss">
   </head>
   <body>
-    <h1>Calendar</h1>
+    <main class="calendar">
+      <ul class="calendar__list calendar__list--month-length--30gi calendar__list--start-day-sun">
+        <li class="calendar__days"></li>
```

Comment:
It should be a singular noun
```suggestion
        <li class="calendar__day"></li>
```